patches/chromium/feat_corner_smoothing_css_rule_and_blink_painting.patch

```diff
@@ -154,23 +89,17 @@ index 4f13354776da9944ed75874953a2e2c9878a649b..83728e697c5029a79fb75d84a1be5bdf
        return a.EmptyCells() == b.EmptyCells();
      case CSSPropertyID::kFill:
 diff --git a/third_party/blink/renderer/core/css/properties/longhands/longhands_custom.cc b/third_party/blink/renderer/core/css/properties/longhands/longhands_custom.cc
-index dc328e04767437d2d9b6d67d07cb6c9d055773f9..01dde66dbbff0c1bf1e1a52486f1901328b4217b 100644
+index dc328e04767437d2d9b6d67d07cb6c9d055773f9..99094ba175693f46171940563e68004cdc90f279 100644
 --- a/third_party/blink/renderer/core/css/properties/longhands/longhands_custom.cc
 +++ b/third_party/blink/renderer/core/css/properties/longhands/longhands_custom.cc
-@@ -12325,5 +12325,25 @@ const CSSValue* InternalEmptyLineHeight::ParseSingleValue(
+@@ -12325,5 +12325,31 @@ const CSSValue* InternalEmptyLineHeight::ParseSingleValue(
                                           CSSValueID::kNone>(stream);
  }
  
 +const CSSValue* ElectronCornerSmoothing::ParseSingleValue(
 +    CSSParserTokenStream& stream,
 +    const CSSParserContext& context,
 +    const CSSParserLocalContext&) const {
```

Comment:
With the blink runtime flag disabled, is this codepath no longer entered? Just want to confirm whether that's the reason why feature checking is no longer needed here.

Reply:
That's correct 👍 For comparison, other experimental CSS properties rely on the JSON definition file to supply the required flag, instead of the parsing code here.